snuba/migrations/parse_schema.py

```diff
@@ -69,7 +69,7 @@ def merge_modifiers(
         return col_type.set_modifiers(existing_modifiers.merge(modifiers))
 
 
-class Visitor(NodeVisitor):
+class Visitor(NodeVisitor):  # type: ignore
```

Comment:
I think this may not be needed after this: https://github.com/getsentry/snuba/pull/1693 Could you try to check if it is still the case ?

Reply:
It's still needed unfortunately